src/styles/index.css

```diff
@@ -19,6 +19,26 @@
   .clip-arc {
     clip-path: ellipse(51% 90% at 50% 100%);
   }
+
+  .clip-arc-border {
+    position: relative;
+    background-color: black;
+    overflow: hidden;
+    clip-path: ellipse(51% 90% at 50% 100%);
+    z-index: 1;
+  }
+  
+  .clip-arc-border::before {
```

Comment:
@pookmish I added this `clip-arc-border` class so that we can differentiate between the top banner style and the paragraphs. I used the same clip-path to maintain uniformity between the two but it looks like the overlapping ellipse is causing these black lines down the edges. 🤔 I'm going to poke around a bit and follow up with @yvonnetangsu tomorrow if she has any thoughts.

Please do not hesitate to let me know if you have anything in mind that might resolve this issue as well!

![image](https://github.com/SU-SWS/summer-nextjs/assets/34019925/6ea6939f-a2fe-430a-a9e0-e3174db9f534)

Reply:
Hmm. I'll take a look.
I'm don't really see a reason to have the new class in the index file since everything other than the clip path can be done with standard tailwind stuff.

Reply:
I put a PR togeather replacing the classes and adjusting the width to fix your issue. Have a look https://github.com/SU-SWS/summer-nextjs/pull/27